Resources/Prototypes/Corvax/Voice/speech_emote_sounds.yml

```
@@ -66,3 +66,68 @@
       collection: FemaleGasp
     DefaultDeathgasp:
       collection: FemaleDeathGasp
+
+# species
+- type: emoteSounds
+  id: MaleTajaran
+  params:
+    variation: 0.125
+  sounds:
+    Scream:
+      path: /Audio/Corvax/Effects/Tajaran/scream.ogg
+    Laugh:
+      path: /Audio/Corvax/Effects/Tajaran/laugh.ogg
+    Sneeze:
+      path: /Audio/Corvax/Effects/Tajaran/sneeze.ogg
```

У тебя файл называете sneze.ogg, либо меняй название файла либо меняй тут на

```suggestion
      path: /Audio/Corvax/Effects/Tajaran/sneze.ogg
```

---

Исправил название файла